scripts/Crawlers/BaseCrawler.py

```diff
@@ -552,3 +558,22 @@ def _check_dats_present(self, directory):
                         return os.path.join(file_path, subfile_name)
             elif file_name.lower() == "dats.json":
                 return file_path
+
+    def _add_source_data_submodule_if_derivedfrom_conp_dataset(
```

Comment:
for consistency, I'd make this fully snake-case
```suggestion
    def _add_source_data_submodule_if_derived_from_conp_dataset(
```
and then refactor above

Reply:
thank you! Fixed in the next commit